nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -421,10 +443,13 @@ def _get_quantization_target_points(self, model: TModel) -> OrderedDict[TargetPo
         if self._quantization_target_points_to_qconfig:
             return self._quantization_target_points_to_qconfig, self._unified_scale_groups
         backend = get_backend(model)
-        pattern = PatternsManager.get_full_pattern_graph(
-            backend=backend, device=self._target_device, model_type=self._model_type
+        device = self._target_device
+        model_type = self._model_type
+        ignored_patterns = PatternsManager.get_full_ignored_pattern_graph(
+            backend=backend, device=device, model_type=model_type
         )
-        quantizer_setup = self._get_quantizer_setup(nncf_graph, pattern)
+        hw_patterns = PatternsManager.get_full_hw_pattern_graph(backend=backend, device=device, model_type=model_type)
```

Comment:
Since both `ignored_patterns` and `hw_patterns` relates to the `device`, they became both `hardware-related`. So, the naming as `hw_patterns` is not proper. I suggest using something with a `fused` word to avoid collisions.

Reply:
Do you suggest changing HWPatterns to FusedPatterns?

Reply:
Yes. Because `ignored_patterns` are hardware-related too.